pkg/grapicmd/util/fs/exec.go

```diff
@@ -0,0 +1,47 @@
+package fs
+
+import (
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/spf13/afero"
+)
+
+func ListExecutableWithPrefix(fs afero.Fs, prefix string) []string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ListExecutableWithPrefix should have comment or be unexported